xbmc/video/VideoDatabase.cpp

```diff
@@ -3934,6 +4039,46 @@ void CVideoDatabase::GetCast(int media_id, const std::string &media_type, std::v
   }
 }
 
+void CVideoDatabase::GetCast(const std::string& media_ids, const std::string &media_type, std::map<int, CVideoInfoTag>& details)
```

Comment:
What's the purpose of passing around these media_ids strings? Don't you already have them as keys in the `details` map?